graylog2-server/src/main/java/org/graylog2/audit/AuditEventTypes.java

```diff
@@ -72,6 +72,7 @@
     public static final String EXTRACTOR_UPDATE = PREFIX + "extractor:update";
     public static final String GETTING_STARTED_GUIDE_OPT_OUT_CREATE = PREFIX + "getting_started_guide_opt_out:create";
     public static final String GROK_PATTERN_CREATE = PREFIX + "grok_pattern:create";
+    public static final String GROK_PATTERN_TEST = PREFIX + "grok_pattern:test";
```

Comment:
Do we really need an audit event for that?

Typically, we only log modifying actions in the audit log, but not read actions.